xml/en/docs/governance.xml

```diff
@@ -0,0 +1,87 @@
+<!--
+  Copyright (C) F5, Inc.
+  -->
+
+<!DOCTYPE article SYSTEM "../../../dtd/article.dtd">
+
+<article name="NGINX Open Source Governance"
+         link="/en/docs/governance.html"
+         lang="en"
+         rev="1">
+
+
+<section>
+
+<para>
+Open source changed the world.
+</para>
+
+<para>
+NGINX is proud to be part of that change, from our very beginning.
+</para>
+
+<para>
```

Comment:
This para reads:

> NGINX is committed to […]. To achieve this _goal_ we are making […] commitments …

So is "continued involvement in open source" a goal or a commitment? Having two levels of commitments is strange.

I suggest simplifying this to cover all of the remaining content on this page (commitments and principles).

> Our continued involvement in the open source community is encapsulated by our Core Commitments and Principles. These will help the longevity, transparency, and impact of our open source projects.